contentcuration/contentcuration/frontend/channelList/views/ChannelSet/StudioCollectionsTable.vue

```diff
@@ -0,0 +1,345 @@
+          });
+        } else if (option.value === 'delete') {
+          this.collectionToDelete =
+            this.getChannelSet(collectionId) ||
+            this.channelSets.find(c => c.id === collectionId) ||
+            null;
+          if (this.collectionToDelete) this.deleteDialog = true;
+        }
+      },
+
+      confirmDelete() {
+        if (this.collectionToDelete) {
+          this.deleteChannelSet(this.collectionToDelete)
+            .then(() => {
+              this.loadChannelSetList();
+              this.deleteDialog = false;
+              this.collectionToDelete = null;
+              this.$store.dispatch('showSnackbarSimple', this.$tr('collectionDeleted'));
+            })
+            .catch(() => {
+              this.deleteDialog = false;
+              this.collectionToDelete = null;
+              this.$store.dispatch('showSnackbarSimple', this.$tr('deleteError'));
+            });
+        }
+      },
+
+      newChannelSet() {
+        this.$router.push({
+          name: RouteNames.NEW_CHANNEL_SET,
+        });
+      },
+    },
+    $trs: {
+      aboutChannelSetsLink: 'Learn about collections',
+      aboutChannelSets: 'About collections',
+      channelSetsDescriptionText:
+        'A collection contains multiple Kolibri Studio channels that can be imported at one time to Kolibri with a single collection token.',
+      channelSetsInstructionsText:
+        'You can make a collection by selecting the channels you want to be imported together.',
+      channelSetsDisclaimer:
+        'You will need Kolibri version 0.12.0 or higher to import channel collections',
+      cancelButtonLabel: 'Close',
+      addChannelSetTitle: 'New collection',
+      noChannelSetsFound:
+        'You can package together multiple channels to create a collection. The entire collection can then be imported to Kolibri at once by using a collection token.',
+      title: 'Collection name',
+      token: 'Token ID',
+      channelNumber: 'Number of channels',
+      options: 'Options',
+      deleteChannelSetTitle: 'Delete collection',
+      deleteChannelSetText: 'Are you sure you want to delete this collection?',
+      cancel: 'Cancel',
+      edit: 'Edit collection',
+      delete: 'Delete collection',
+      saving: 'Saving',
+      collectionDeleted: 'Collection  deleted',
+      deleteError: 'Error deleting collection',
+    },
+  };
+
+</script>
+
+
+<style lang="scss" scoped>
+
+  .page-container {
+    width: 100%;
+    max-width: 1440px;
+    margin: 0 auto;
+    text-align: center;
+  }
+
+  .table-header {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 16px;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  .header-left,
+  .header-right {
+    display: flex;
+    align-items: center;
+  }
+
+  .info-content p {
+    margin-bottom: 16px;
+  }
+
+  .actions-cell {
+    display: flex;
+    justify-content: flex-end;
+  }
+  @media (max-width: 500px) {
```

Comment:
We avoid CSS media queries in favor of our layout system - mostly for consistency. Can you use `useKResponsiveWindow`?

- https://design-system.learningequality.org/layout#responsiveness
- https://design-system.learningequality.org/usekresponsivewindow/